src/Makefile.am

```diff
@@ -543,6 +563,7 @@ squid_LDADD = \
 	DiskIO/libdiskio.la \
 	comm/libcomm.la \
 	anyp/libanyp.la \
+	error/liberror.la \
```

Comment:
libsecurity and SSL_LIBS content depends on liberror yes? That means liberror should be listed later.

Reply:
commit 0b25e37359